lib/wallet/account/ledger-account.ts

```diff
@@ -139,11 +139,15 @@ export class LedgerAccount implements IWallet {
 
   public async signAndBroadcastTx(
     outputs: PaymentOutput[],
-    options?: TransactionOptions,
+    options?: TxSettingOptions,
   ): Promise<string> {
     throw new Error("Method not implemented.");
   }
 
+  public async signTx(tx: Transaction): Promise<Transaction> {
```

Comment:
Currently, we don't ensure how to let ledger sign transaction inputs then create signatures for them. Keep `not implemented` error here.